examples/03_connectivity/plot_probabilistic_atlas_extraction.py

```diff
@@ -62,8 +62,10 @@
 from nilearn import plotting
 # Mask out the major diagonal
 np.fill_diagonal(correlation_matrix, 0)
+# We reorder the correlation matrix to make the block structure
+# of invidiual regions more visible.
 plotting.plot_matrix(correlation_matrix, labels=labels, colorbar=True,
-                     vmax=0.8, vmin=-0.8)
+                     vmax=0.8, vmin=-0.8, reorder=True)
```

Comment:
The MSDL atlas is actually ordered by network, so I am not sure that this is the best example to add reordering :)

Reply:
I would try on the following example:
http://nilearn.github.io/auto_examples/03_connectivity/plot_signal_extraction.html

The ordering of the matrix is a bit ugly.

Reply:
Okay! Worked well, so I added it to this example.